### DISCUSSION TITLE
Github feed is too slow

### DISCUSSION THREAD

**Discussion by eldoy** [community/community#68722] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I can't wait 5 seconds to see my feed every time I go to my Dashboard! How is it this slow in 2023? Most of the time it's the same feed as well! Have you heard about caching at all over there at Github? The feed must arrive within 0.1 seconds, or it's useless and annoying. This is terrible.

**Comment by sherloook2** [comment]
hello 

I completely agree with this concern. Performance is a critical aspect of the user experience, especially in 2023, where efficiency and speed have become the standard.

The idea of caching the GitHub feed seems like a smart approach to significantly improve loading times. A feed that loads in less than 0.1 seconds would not only be practical but also highly appreciated by the user community.

I hope this suggestion will be taken into consideration and contribute to making GitHub even more efficient and enjoyable for everyone.
